release.json

```diff
@@ -1,7 +1,7 @@
 {
   "nio/NioCommand": {
     "language": "Python",
-    "version": "0.0.1",
-    "reference": "github.com/nio-blocks/nio_command.git@v0.0.1"
+    "url": "git@github.com:nio-blocks/nio_command.git",
```

Comment:
I think if we're using this format for now, the version tag at the end still needs to be there